CTOR-897 : Refresh network-cisco-firepower-snmp (#3562)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **Cisco Firepower SNMP** brings a host template:

* **Net-Cisco-Firepower-SNMP-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="Net-Cisco-Firepower-SNMP-custom" label="Net-Cisco-Firepower-SNMP-custom">

| Service Alias | Service Template                         | Service Description                         |
|:--------------|:-----------------------------------------|:--------------------------------------------|

> | Cpu           | Net-Cisco-Firepower-Cpu-SNMP-custom      | Check the rate of the utilization of CPU    |

Reviewer: ```suggestion
| Cpu           | Net-Cisco-Firepower-Cpu-SNMP-custom      | Check the rate of the utilization of the CPU    |
```

| Faults        | Net-Cisco-Firepower-Faults-SNMP-custom   | Check faults                                |
| Hardware      | Net-CIsco-Firepower-Hardware-SNMP-custom | Check hardware environment                  |

> | Memory        | Net-Cisco-Firepower-Memory-SNMP-custom   | Check the rate of the utilization of memory |

Reviewer: ```suggestion
| Memory        | Net-Cisco-Firepower-Memory-SNMP-custom   | Check the rate of the utilization of the memory |
```

|:----------------------------------------------------------|:------|
| *interface_name*#status                                   | N/A   |
| *interface_name*#interface.traffic.in.bitspersecond       | b/s   |
| *interface_name*#interface.traffic.out.bitspersecond      | b/s   |
| *interface_name*#interface.packets.in.discard.percentage  | %     |
| *interface_name*#interface.packets.in.error.percentage    | %     |
| *interface_name*#interface.packets.out.discard.percentage | %     |
| *interface_name*#interface.packets.out.error.percentage   | %     |

</TabItem>
<TabItem value="Memory" label="Memory">

| Metric name                      | Unit  |
|:---------------------------------|:------|
| *memory*#memory.usage.bytes      | B     |
| *memory*#memory.free.bytes       | B     |
| *memory*#memory.usage.percentage | %     |

</TabItem>
</Tabs>

## Prerequisites

### SNMP Configuration

The SNMP service must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.

### Network flow

The target resource must be reachable from the Centreon poller on the UDP/161
SNMP port.

### Configuring Cisco Firepower equipment

> To control your Cisco Firepower, the SNMP must be configured (see official documentation: https://www.cisco.com/c/en/us/support/docs/ip/simple-network-management-protocol-snmp/213971-configure-snmp-on-firepower-ngfw-applian.html?dtid=osscdc000283).

Reviewer: ```suggestion
To monitor your Cisco Firepower, SNMP must be configured (see official documentation: https://www.cisco.com/c/en/us/support/docs/ip/simple-network-management-protocol-snmp/213971-configure-snmp-on-firepower-ngfw-applian.html?dtid=osscdc000283).
```

Reviewer: le mettre au-dessus, c'est la même chose que la 1è sous-partie
